src/pages/calendar/containers/Content.tsx

```diff
@@ -57,31 +57,55 @@ export function Content({ term }: ContentProps): JSX.Element {
           </Alert>
         )}
         <Flex
+          justifyContent={'space-between'}
           justifyItems="center"
-          alignItems={{ base: 'start', sm: 'center' }}
+          alignItems={{ base: 'start', md: 'center' }}
           direction={{ base: 'column', sm: 'row' }}
+          gap={{ base: '5', md: '10' }}
         >
-          <Skeleton isLoaded={!loading} display="flex" flexDirection="row" alignItems="center">
+          <Skeleton isLoaded={!loading} display="flex" flexDirection="column" alignItems="flex-start">
             {data && (
               <>
-                <Heading mr="5" size="2xl" as="h2" whiteSpace="pre">{`${data.subject} ${data.code}`}</Heading>
-                <Heading size="lg" as="h3" color={mode('gray', 'dark.header')}>
+                <Heading mr="5" size="3xl" fontSize={{ base: '3xl', xl: '5xl' }} as="h2" whiteSpace="pre">
+                  {`${data.subject} ${data.code}`}
+                </Heading>
+                <Heading
+                  size="lg"
+                  fontSize={{ base: 'lg', md: '2xl', xl: '3xl' }}
+                  as="h3"
+                  color={mode('gray', 'dark.header')}
+                >
                   {data.title}
                 </Heading>
               </>
             )}
           </Skeleton>
-          <Spacer />
           {!error && (
-            <Button
-              rightIcon={courseIsSaved ? <MdDelete /> : <MdAdd />}
-              onClick={handleBookmarkClick}
-              colorScheme={courseIsSaved ? 'red' : 'green'}
-              size="sm"
-              disabled={loading}
-            >
-              {courseIsSaved ? 'Remove from Timetable' : 'Add to Timetable'}
-            </Button>
+            <>
+              <Button
```

Comment:
You can leave the button as is for now, I'm making it responsive with some context relevant to the PR you mentioned. Since the page itself isn't responsive yet (until that PR goes up) it's not a big deal to stay the same and get squishy